lib/spec.gb

```diff
@@ -68,6 +78,15 @@ class Describe
   def run_specs
     puts " " * @indentation + context_name.to_s
     examples.each do |example_node|
+      @before_hooks.each do |before_hook|
+        before_hook.call
+      end
+
+      example_node.run
+
+      @after_hooks.each do |after_hook|
```

Comment:
Is this modeled after RSpec? If so, RSpec has different semantics:

> after hooks are guaranteed to run even when there are exceptions in before hooks or examples. When an exception is raised in an after block, the exception is captured for later reporting, and subsequent after blocks are run.

But I'm not familiar with the design of Goby's testing library.

Reply:
I'd like to do that as well, but Goby doesn't have the capability to capture exceptions yet 😬

Reply:
Ach!